brion/morphology.h

```diff
@@ -132,11 +156,22 @@ class Morphology : public boost::noncopyable
      *
      * @param apicals section and point index of all apical points
      * @throw std::runtime_error if object not created with write ctor
-     * @throw std::runtime_error if called for version 1 files
+     * @throw std::runtime_error if not supported by implementation
```

Comment:
But if you deprecate the constructor to open a v2 morphology for writing, then you are effectively deprecating this function also.